mtbf_driver/mtbf.py

```diff
@@ -226,6 +227,8 @@ def start_gaiatest(self):
             self.passed = self.runner.passed + self.passed
             self.failed = self.runner.failed + self.failed
             self.todo = self.runner.todo + self.todo
+            if self.marionette:
```

Comment:
We might not keep marionette instance since marionette server issue.
Perhaps can change to use self.runner.marionette